qpc/report/__init__.py

```diff
@@ -17,10 +17,12 @@
 DEPLOYMENTS = 'deployments'
 INSIGHTS = 'insights'
 MERGE = 'merge'
+UPLOAD = 'upload'
 MERGE_STATUS = 'merge-status'
 DOWNLOAD = 'download'
 REPORT_URI = '/api/v1/reports/'
 DETAILS_PATH_SUFFIX = '/details/'
 DEPLOYMENTS_PATH_SUFFIX = '/deployments/'
 INSIGHTS_PATH_SUFFIX = '/insights/'
 ASYNC_MERGE_URI = '/api/v1/reports/merge/jobs/'
+REPORT_UPLOAD_URI = '/api/v1/reports/'
```

Comment:
Do we need to add this variable since it is the same as the `REPORT_URI`?

Reply:
Removed